src/utils/utils.js

```diff
@@ -30,3 +30,11 @@ export var colorLuminance = function(hex, lum) {
 
   return rgb;
 };
+
+/*
+ * check if variable is function type. http://stackoverflow.com/questions/5999998/how-can-i-check-if-a-javascript-variable-is-function-type
+ */
+export var isFunction = function(functionToCheck) {
+    var getType = {};
+    return functionToCheck && getType.toString.call(functionToCheck) === '[object Function]';
```

Comment:
why did you pick this method instead if shorter `if (typeof functionToCheck === "function") {` (most popular answer in your link)?